firefox_ui_tests/firefox_ui_tests/functional/security/test_safe_browsing_primary_download.py

```diff
@@ -0,0 +1,96 @@
+import os
+import sys
+
+from marionette_driver import By, Wait
+
+from firefox_puppeteer.testcases import FirefoxTestCase
+
+from firefox_puppeteer.ui.browser.window import BrowserWindow
```

Comment:
Whereby I don't see that BrowserWindow is used at all. So you can remove line 8.